src/s390_crypto.c

```diff
@@ -695,6 +724,7 @@ int s390_get_functionlist(libica_func_list_element *pmech_list,
 		pmech_list[x].mech_mode_id != RSA_KEY_GEN_CRT)
 		pmech_list[x].flags &= ~ICA_FLAG_SW;
 #endif
+
```

Comment:
Empty line added. Not sure if this is intentional.....

Reply:
Empty line is removed with the fix for EC keygen sw flag, see comment below.